index.js

```diff
@@ -76,6 +77,20 @@ module.exports = function (str, opts) {
 		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
 	}
 
+	// remove directory index
+	if (opts.removeDirectoryIndex === true) {
+		opts.removeDirectoryIndex = [/^index/];
```

Comment:
The regex is too loose. This will match even `indexer-for-unicorn.php`. I think we should use `/^index\.[a-z]$/` as default. Same with the `default` example in the readme.